### ISSUE TITLE
IWP: Writer Bio Content Type

### ISSUE THREAD

**Issue by makurj** [uiowa/uiowa#7282]
For IWP's D9 site, create a custom content type to match https://iwp.uiowa.edu/node/add/public-writer-bio. 

- This content type will be used to filter authors by year, country and language so those fields should be exposed, if creating a view. 
- We would also like it if the filters could be "and" instead of "or". For example, show me writers from Canada AND in year 2005 AND language French.
- Run list of countries by taxonomy so we can add/edit. Remove "regions" field and any regions from the country/region list.
- Remove these fields from writer bio: gender, name suffix, phonetic, pseudonym.
- Change “given name” to “preferred name”.

Sample: https://iwp.uiowa.edu/residency/participants

If possible, migrate all bios programmatically from D7 to D10. Make sure special characters come over correctly.

**Comment by makurj** [comment]
Make sure these countries are represented in addition to Germany for the ["Regions and Countries" field](https://iwp.uiowa.edu/node/add/public-writer-bio) and for the view dropdown: https://iwp.uiowa.edu/residency/participants-by-region/Germany:

German Democratic Republic (DDR)
German Federal Republic (BRD)
Germany

![image](https://github.com/uiowa/uiowa/assets/52135878/41a9eb74-2264-42b3-9a7f-6a1c781dd4b4)
![image](https://github.com/uiowa/uiowa/assets/52135878/7ad3e406-001a-40fa-8df1-c19dfa06fc94)

**Comment by joewhitsitt** [comment]
@makurj and @briand44 can this be closed?

**Comment by briand44** [comment]
Yeah, work was done in https://github.com/uiowa/uiowa/issues/7358
